tests/test_reloader.py

```diff
@@ -23,16 +23,20 @@
 except ImportError:
     flags = 0
 
+
 def terminate(proc):
     if flags:
         proc.send_signal(CTRL_BREAK_EVENT)
     else:
         proc.terminate()
 
+
 def write_app(filename, **runargs):
     text = secrets.token_urlsafe()
     with open(filename, "w") as f:
-        f.write(dedent(f"""\
+        f.write(
+            dedent(
+                f"""\
```

Comment:
I suppose this was by linter because the original formatting was better. Maybe reindent the rest of the string literal too to match the extra level at opening?

Reply:
Yes. `make fix-import`

Oh trust in the almighty `black`.